pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <properties>
 			<openmrs.platform.version>2.4.2</openmrs.platform.version>
-			<openmrs.fhir2.version>1.4.0</openmrs.fhir2.version>
+			<openmrs.fhir2.version>1.5.0</openmrs.fhir2.version>
```

Comment:
does fhir2 has released 1.5.0 version? 
I think today only they have released this version :) https://talk.openmrs.org/t/announcing-fhir2-1-5-0/37372
@binduak I hope all required changes we are looking in 1.5.0-SNAPSHOT are included in this version.